src/components/register.js

```diff
@@ -180,7 +180,7 @@ const RegistrationForm = (props) => {
               </button>
             </Form.Item>
           </Form>
-          <div id='or_login'>
+          <div id='or_login' data-test-id="loginLinkWrapper">
```

Comment:
Everything looks great, but the test fails at first since your `data-test-id` needs to be a `data-testid` instead, just a simple mistype, but after you get that changed you should be good to go! Nice work so far Ben!

Reply:
Whoops! Thanks for pointing this out. Should be fixed now as well as the merge conflicts.